src/main/java/cz/fi/muni/pa165/esports/dao/TeamDao.java

```diff
@@ -0,0 +1,19 @@
+package cz.fi.muni.pa165.esports.dao;
+
+import cz.fi.muni.pa165.esports.entity.Team;
+
+import java.util.List;
+
+/**
+ * @author Gabriela Kandova
+ */
+
+public interface TeamDao {
+    void create(Team team);
+    void delete(Team team);
+
+    Team findById(Long id);
+    List<Team> findAll();
+    Team findByName(String name);
```

Comment:
So, did we agree that the team name is unique? because maybe when searching for a name, there might be multiple results matching